srml/assets/Cargo.toml

```diff
@@ -2,18 +2,26 @@
 name = "srml-assets"
 version = "0.1.0"
 authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
 
 [dependencies]
 hex-literal = "0.1.0"
 serde = { version = "1.0", default-features = false }
 parity-codec = { version = "3.0", default-features = false }
+# Needed for deriving `Encode` and `Decode` for `RawEvent`.
```

Comment:
I'm not sure that it makes sense to preserve these comments.

Reply:
Could you fix `decl_event` so that we not require adding this for each crate?

Reply:
I'm going to try.

Reply:
@bkchr #1663 should do the trick.

Reply:
Could you please just merge #1663. We don't need an extra pr for each crate. Ty :)

Reply:
@bkchr Sorry, too late. Though I have updated this pull request and excluded needless dependencies.